### PR TITLE
Hterm

### PR DESCRIPTION
**Pull request by Izak88**

### CODE REVIEW

File: src/api/docker.ts [hunk 1]
```diff
@@ -104,7 +103,11 @@ export function attachExec(id: string, cmd: any): Observable<any> {
         } else if (str.includes('[success]')) {
           exitCode = 0;
           ws.end();
-        } else if (!str.includes('/usr/bin/abstruse') && !str.startsWith('>')) {
+        } else if (!str.includes('/usr/bin/abstruse \'' + cmd.command) && !str.startsWith('>')) {
+          if (str.includes('//') && str.includes('@')) {
```
Comment (jkuri) [inline src/api/docker.ts]: isn't this already on inside `process.ts`?

Comment (Izak88) [reply src/api/docker.ts]: No. In `process.ts` there is just `observer.next(event);`.

Comment (jkuri) [reply src/api/docker.ts]: its in config.ts, I knew I saw this somewhere :)

Comment (Izak88) [reply src/api/docker.ts]: In `config.ts`, an access token is added to cloneUrl, here its the opposite, here is token removed from a string, so it's not shown in console output.

**Pull request closed without merging**